modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/FragmentPreview.es.js

```diff
@@ -160,7 +160,7 @@ class FragmentPreview extends PortletBase {
 		if (!this._loading) {
 			this._loading = true;
 
-			this.fetch(this.urls.render, {
+			fetch(this.urls.render, {
 				configuration: this.configuration,
 				css: this.css,
 				html: this.html,
```

Comment:
We should send fragments and segments as a separate PR; fragments is Echo team and segments Tango team. This ticket, LPS-98949, can be for segments.